docker/push.sh

```diff
@@ -21,3 +21,14 @@ echo "Pushing image to DockerHub"
 docker push $image
 
 echo "Image pushed $image"
+
+/usr/bin/env ruby <<-EORUBY
+  require "erb"
+  require "yaml"
+
+  image = "`echo $image`"
```

Comment:
`image = ENV["image"]` :)

Reply:
Also, please make sure that the script fails if this variable is not exported.

Reply:
Right, I'll export it and then `ENV.fetch("image")` which would throw me an error if it isn't set.